# Задача

Є можливість вибрати спосіб виробництва і реалізації двох наборів товарів широкого вжитку. За даними відділу маркетингу, яким були проведені дослідження ринку, можливий прибуток від виробництва і реалізації X і Y наведено в таблицях (X,Y  – прибуток в грощових одиницях).

X   |1000 |1500 |2000|
----|:---:|:---:|---:|
P   |0,5  |0,3  |0,2 |

Y   |1000 |1500 |1750|
----|:---:|:---:|---:|
P   |0,4  |0,4  |0,2 |

Потрібно оцінити ступінь ризику і запропонувати рішення щодо випуску і реалізації одного із набора товарів.

# Розв’язок

Спочатку визначимо математичне очікування для кожного набору товарів:

$M[X] = \sum_1^3{X_i∙P_i}$ = 1000∙0,5 + 1500∙0,3 + 2000∙0,2 = 



In [7]:
1000*0.5 + 1500*0.3 + 2000*0.2

1350.0

$M[Y] = \sum_1^3{Y_i∙P_i}$ = 1000∙0,4 + 1500∙0,4 + 1750∙0,2 = 

In [2]:
1000*0.4 + 1500*0.4 + 1750*0.2

1350.0

Визначимо дисперсію для кожного набору товарів.


$D[X] = M[(X-M[X])^2] = 0,5∙(1000-1350)^2+0,3∙(1500-1350)^2+0,2∙(2000-1350)^2=$

In [6]:
0.5*(1000-1350)**2+0.3*(1500-1350)**2+0.2*(2000-1350)**2

152500.0

$D[Y] = M[(Y-M[Y])^2] = 0,4∙(1000-1350)^2+0,4∙(1500-1350)^2+0,2∙(1750-1350)^2=$

In [8]:
0.4*(1000-1350)**2+0.4*(1500-1350)**2+0.2*(1750-1350)**2

90000.0

Визначимо середньоквадратичне відхилення для кожного набору товарів. 

$σ_X = \sqrt {D[X]} = $

In [13]:
round(152500.0**0.5, 3)

390.512

$σ_Y = \sqrt {D[Y]} = $

In [14]:
round(90000.0**0.5, 3)

300.0

# Відповідь
$σ_X = 390.512$

$σ_Y = 300.0$

$σ_Y < σ_X$

Оскільки середньоквадратичне відхилення для 𝑌 менше за середньоквадратичне відхилення для 𝑋, то випуск товарів з набору 𝑌 є більшщ доцільним.